gamemode/modules/base/cl_fonts.lua

```diff
@@ -2,22 +2,28 @@
 The fonts that DarkRP uses
 ---------------------------------------------------------------------------]]
 local function loadFonts()
-    local tahoma = system.IsLinux() and "DejaVu Sans" or "Tahoma"
-    local tahomaSize = system.IsLinux() and fp{fn.Flip(fn.Add), 1} or fn.Id
+    local robotoSize = system.IsLinux() and fp{fn.Flip(fn.Add), 1} or fn.Id
```

Comment:
Does the hack that was applied for Tahoma really also apply for Roboto?

Reply:
No it does not.
Roboto may look different on linux however, I can't know.